src/components/Nav/Nav.js

```diff
@@ -1,6 +1,74 @@
-import React from 'react';
+import React, { useState } from 'react';
+import NavAside from './NavAside/NavAside.js';
+import NavDropdown from './NavDropdown/NavDropdown.js';
+import * as S from './NavStyle.js';
 
 function Nav() {
-  return <h1>Nav 입니다.</h1>;
+  const [isDropdownHover, setIsDropdownHover] = useState(false);
+  const [searchValue, setSearchValue] = useState('');
+
+  const haddleInput = event => {
+    setSearchValue(event.target.value);
+  };
+
+  return (
+    <>
+      <S.Nav onMouseLeave={() => setIsDropdownHover(false)}>
+        <S.NavTopContainer>
+          <S.NavLogo>
+            <S.NavLogoImage src="images/logo.png" />
+          </S.NavLogo>
+          <S.NavSearchContainer>
+            <S.NavSearch
+              placeholder="상품, 지역명 검색"
+              onChange={e => haddleInput(e)}
```

Comment:
```suggestion
              onChange={haddleInput}
```
이벤트 객체는 기본값으로 전달됩니다!